cmd/helm/search_hub.go

```diff
@@ -51,16 +59,16 @@ func newSearchHubCmd(out io.Writer) *cobra.Command {
 	o := &searchHubOptions{}
 
 	cmd := &cobra.Command{
-		Use:   "hub [keyword]",
-		Short: "search for charts in the Helm Hub or an instance of Monocular",
+		Use:   "hub [KEYWORD]",
+		Short: "search for charts in the Artifact Hub or your own hub instance",
 		Long:  searchHubDesc,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return o.run(out, args)
 		},
 	}
 
 	f := cmd.Flags()
-	f.StringVar(&o.searchEndpoint, "endpoint", "https://hub.helm.sh", "monocular instance to query for charts")
+	f.StringVar(&o.searchEndpoint, "endpoint", "https://hub.helm.sh", "Hub instance to query for charts")
```

Comment:
Is this still the correct endpoint?

Reply:
Yes, but only because in the helm dev meeting where I brought this up the group decided to keep the helm.hub.sh domain. The stated reason was if we ever wanted to change the default hub again, we could retroactively do this for previous client versions as well since we control the domain. fe41873 was made as a concession to this decision. It works because for the foreseeable future it's a passthrough to artifacthub.io, and artifact hub has a redirect in place for the monocular paths to the artifact hub chart paths.

Reply:
@jdolitsky To add to what @scottrigby had to say... Artifact Hub is a sandbox project. If something happens to it and it does not continue on long term we need a method to not break users. By controlling the DNS we could take it back and users would not be affected. This is a contingency just in case. Currently, Artifact Hub manages hub.helm.sh so that it can point to their things.